### PR TITLE
I think I fixed everything

### PR DESCRIPTION
**Pull request by RedPrez16**

### CODE REVIEW

File: websites/G/GetPocket/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  },
+  "url": [
+    "www.getpocket.com",
+    "getpocket.com",
+    "app.getpocket.com"
+  ],
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/7Qre9TD.png",
+  "thumbnail": "https://i.imgur.com/ac3w7ln.png",
+  "color": "#12171C",
```
Comment (Bas950) [inline websites/G/GetPocket/dist/metadata.json:19]: thumbnail should be a promotional banner or a screenshot of the website homepage

**Review by Bas950 — CHANGES_REQUESTED** [review]
Upload proof of it working
Fix codacy errors

File: websites/G/GetPocket/dist/metadata.json (hunk 1 above)
Comment (RedPrez16) [reply websites/G/GetPocket/dist/metadata.json]: ProtonMail? Why? It's not ProtonMail. It's Get Pocket.

**Comment by RedPrez16** [comment]
Okay

**Comment by ririxidev** [comment]
oh bruh i merged different pr 🤦

**Comment by RedPrez16** [comment]
Is mine good though?

**Comment by ririxidev** [comment]
Oh no okey, it didn't merge any files so you need to create another pr :)

**Comment by RedPrez16** [comment]
Okay. Let me figure out how to do that on mobile.

**Comment by Bas950** [comment]
and this is why I hate pull bot xD
its not hard to keep your fork up to date yourself so idk why you even use it lmao

**Comment by RedPrez16** [comment]
What do you mean?

**Comment by RedPrez16** [comment]
Oh, I don't know how GitHub works. So I have no idea how to update it.

**Comment by Bas950** [comment]
If you use github desktop its 3 clicks:
https://i.imgur.com/5wv15Kc.gifv

**Comment by RedPrez16** [comment]
Oh, okay. Thank you. Also, do you see my presence on my fork? As I don't.

**Pull request merged**